README.md

```diff
@@ -59,6 +61,38 @@ NOTE: `VULKAN_LOADER_GENERATED_DIR` should be the directory that contains `vk_la
 
 See [docker/build.Dockerfile](docker/build.Dockerfile) for detailed Ubuntu build instructions.
 
+## Enabling the layers:
+To enable the performance layers, set these environent variables:
```

Comment:
These steps are linux-specific. What do you think about saying something like this: `To enable performance layers on linux, set the environment variables listed below. For other platforms, see the article layer configuration <link>.`?